src/sentry/runner/commands/presenters/optionspresenter.py

```diff
@@ -0,0 +1,21 @@
+from abc import ABC, abstractmethod
+
+
+class OptionsPresenter(ABC):
```

Comment:
Also this is an interface with multiple implementations, please add a docstring that explains the purpose of the class